jobs/quoscient_cybersecurity_germany_permanent.html

```diff
@@ -0,0 +1,57 @@
+---
+title: Senior Python Developer
+company: QuoScient GmbH
+url: http://quoscient.io
+location: Frankfurt am Main, Germany
+contract: permanent
+contact:
+    name: Ina Klemm
+    email: careers@quoscient.io
+created: !!timestamp '2017-03-01' 
+tags:
+  - germany
+  - python 3
```

Comment:
Everything on this board is Python-oriented - you don't need to provide that as a keyword. I don't think Python 3 is all that unusual a technology these days so I'd recommend that you remove this.